cmd/test.go

```diff
@@ -249,6 +250,12 @@ func (t *Test) GetCmd() *cobra.Command {
 			path += "/keploy"
 
 			testReportPath := path + "/testReports"
+			subDirPrefix := "test-run-"
```

Comment:
Please define this as constant for the template of test-run reports in the models package.

Reply:
This will in maintaining the template from one place.